app/src/main/java/io/gnosis/safe/ui/settings/safe/AdvancedSafeSettingsFragment.kt

```diff
@@ -91,14 +91,11 @@ class AdvancedSafeSettingsFragment : BaseViewBindingFragment<FragmentSettingsSaf
                 openable = false
                 name = getString(R.string.safe_settings_not_set)
             }
-            viewModel.isDefaultFallbackHandler(fallbackHandler) -> labeledAddress(fallbackHandler, R.string.safe_settings_default_fallback_handler)
-            else -> labeledAddress(fallbackHandler, R.string.safe_settings_unknown)
+            viewModel.isDefaultFallbackHandler(fallbackHandler) -> labeledAddress(fallbackHandler, io.gnosis.data.R.string.default_fallback_handler)
```

Comment:
why full package path?

Reply:
I didn't know its not necessary for module. But it is not necessary.

Reply:
I assumed I would need it in any file that also uses apps R. But apparently only SafeRepository needs this import: 
`import io.gnosis.data.R`